BitwardenShared/UI/Platform/Application/Views/FloatingActionButton.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+// MARK: - FloatingActionButton
+
+/// A view representing a floating action button for adding new items.
+///
+struct FloatingActionButton: View {
+    // MARK: Properties
+
+    /// The bottom content offset padding around the floating action button.
+    /// This is the button size plus the bottom/top padding (50 + 16 + 16).
+    static let bottomOffsetPadding: CGFloat = 82
```

Comment:
❓ It looks like this is assuming the button is 50x50, but it looks like the image is 24x24 with 12 pt of padding = 48x48. Would it be better to give the button a fixed frame and not specify the padding?